tests/integration/test_cloudformation.py

```diff
@@ -1780,6 +1780,17 @@ def test_cfn_with_multiple_route_table_associations(self, ec2_client, deploy_cfn
         # CloudFormation will create more than one route table 2 in template + default
         assert len(route_table["Associations"]) == 3
 
+        # assert subnet attributes are present
```

Comment:
Maybe we can use this opportunity to make this test `aws_validated`? :thinking:

Reply:
Good point - will make sure to do that in a follow-up iteration. 👍